### ISSUE TITLE
CORE-AAM mapping `aria-haspopup=true` for ATK/ATSPI and IA2 in conflict with the spec?

### ISSUE THREAD

**Issue by spectranaut** [w3c/core-aam#189]
In the spec for [aria-haspopup](https://w3c.github.io/aria/#aria-haspopup), it says:
> To provide backward compatibility with ARIA 1.0 content, user agents MUST treat an aria-haspopup value of true as equivalent to a value of menu.

But in CORE-AAM, the mappings pass through `true` as `true`: https://w3c.github.io/core-aam/#ariaHaspopupTrue

Thanks to @sivakusayan and @jcsteh for catching this, really.

**Comment by scottaohara** [comment]
that just seems like it should be the other way around - since aria-haspopup=true was the original value and menu was the value added to 'replace' it

**Comment by jcsteh** [comment]
It looks like Chrome maps aria-haspopup="true" to "menu" for IA2 thanks to [this cross-platform code](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/modules/accessibility/ax_node_object.cc;l=3619;drc=f2732aef3e95dca51f5d70c1242f0c7614a0b9aa;bpv=0;bpt=1). (Thanks to @spectranaut  for the code link.) I haven't tested ATK, but I imagine it would be the same.

**Comment by jcsteh** [comment]
Firefox currently maps aria-haspopup="true" to "true" for both IA2 and ATK. Given that Firefox currently breaks Mac, it probably makes the most sense to fix this in the spec across the board and update Firefox accordingly.

**Comment by spectranaut** [comment]
@sivakusayan I can't assign this issue to you because you aren't a member of the W3C, but it's you issue in spirit until you say you are not longer interested :)
